frontends/infinite-corridor/webpack.config.js

```diff
@@ -3,10 +3,8 @@ const path = require("path")
 const webpack = require("webpack")
 const BundleTracker = require("webpack-bundle-tracker")
 const MiniCssExtractPlugin = require("mini-css-extract-plugin")
-const CKEditorWebpackPlugin = require("@ckeditor/ckeditor5-dev-webpack-plugin")
 const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer")
-
-const { styles } = require("@ckeditor/ckeditor5-dev-utils")
+const { withCKEditor } = require("ol-ckeditor/webpack-utils")
```

Comment:
CKEditor requires some special webpack configuration. Almost all of that has been moved to `withCKEditor`.